build/yamls/base/security-crds.yml

```diff
@@ -119,3 +119,124 @@ spec:
                            cidr:
                              type: string
                              format: cidr
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusternetworkpolicies.security.antrea.tanzu.vmware.com
+spec:
+  group: security.antrea.tanzu.vmware.com
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+  scope: Cluster
+  names:
+    plural: clusternetworkpolicies
+    singular: clusternetworkpolicy
+    kind: ClusterNetworkPolicy
+    shortNames:
+      - cnp
+  # Prune any unknown fields
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      properties:
+        spec:
+          # Ensure that Spec.AppliedTo and Spec.Priority fields are set
+          required:
+            - appliedTo
+            - priority
+          type: object
+          properties:
+            priority:
+              type: number
+              format: float
+              # Ensure that Spec.Priority field is between 1 and 10000
+              minimum: 1.0
+              maximum: 10000.0
+            appliedTo:
+              type: array
+              items:
+                type: object
+                # Ensure that Spec.AppliedTo does not allow IPBlock field
+                properties:
+                  podSelector:
+                    type: object
+                    additionalProperties:
+                      type: object
+                  namespaceSelector:
+                    type: object
+                    additionalProperties:
+                      type: object
```

Comment:
did we have a conversation last week about this being incorrect (because of a missing `x-kubernetes-preserve-unknown-fields`), or was it something else?

Reply:
That's weird. I thought I updated this. Maybe lost in merge again. Let me add that back.